new/LIT5993Hymn.xml

```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5993Hymn" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Hymn to Mary</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2020-05-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <div type="textpart" subtype="incipit">
+                    <note>The following incipit is taken from <ref type="mss" corresp="BDLaethd20"/>.</note>
+                    <ab>
+                       ኦእግዝእትነ፡ ቅድስት፡ ድንግል፡ በ፪ማርያም፡ ማ<sic resp="DR">ረ</sic>ሃም፡ እሙ፡ ለኢየሱስ፡ ክርስቶስ፡ ከሀሉ፨
```

Comment:
1. Couldn't ከሀሉ፡ be ከሀሊ፡ (i.e., properly, ከሃሊ፡ "almighty") ?
2. In that case, the incipit looks like that of LIT3093RepCh373. The latter is found, among other mss, in ms Ethio-SPaRE DMB-003 (ESdmb003): I can see there on f. 9rb-va that both the incipit and the explicit are the same that the those here encoded (with some minor variants, such as ሃ in the name of Mary). I would be in favour of not creating this record and using LIT3093RepCh373 for the hymn in BDLaethd20. What do you think?

Reply:
You're right, it's the same. Wonderful, thank you!